runtime/centrifuge/src/lib.rs

```diff
@@ -418,6 +429,92 @@ impl orml_asset_registry::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub LiquidityPoolsPalletIndex: PalletIndex = <LiquidityPools as PalletInfoAccess>::index() as u8;
+}
+
+impl pallet_liquidity_pools::Config for Runtime {
+	type AccountConverter = AccountConverter<Runtime>;
+	type AdminOrigin = EnsureRootOr<TwoThirdOfCouncil>;
+	type AssetRegistry = OrmlAssetRegistry;
+	type Balance = Balance;
+	type CurrencyId = CurrencyId;
+	type ForeignInvestment = Investments;
+	type GeneralCurrencyPrefix = cfg_primitives::liquidity_pools::GeneralCurrencyPrefix;
+	type OutboundQueue = WrapperOutboundQueue;
+	type Permission = Permissions;
+	type PoolId = PoolId;
+	type PoolInspect = PoolSystem;
+	type Rate = Rate;
+	type RuntimeEvent = RuntimeEvent;
+	type Time = Timestamp;
+	type Tokens = Tokens;
+	type TrancheCurrency = TrancheCurrency;
+	type TrancheId = TrancheId;
+	type TrancheTokenPrice = PoolSystem;
+	type TreasuryAccount = TreasuryAccount;
+	type WeightInfo = ();
+}
+
+type LiquidityPoolsMessage =
+	pallet_liquidity_pools::Message<Domain, PoolId, TrancheId, Balance, Rate>;
+
+/// The WrapperOutboundQueue serves as a filter for outbound LP messages that we
+/// want to allow initially.
+pub struct WrapperOutboundQueue;
+
+impl OutboundQueue for WrapperOutboundQueue {
+	type Destination = Domain;
+	type Message = LiquidityPoolsMessage;
+	type Sender = AccountId;
+
+	fn submit(
+		sender: Self::Sender,
+		destination: Self::Destination,
+		msg: Self::Message,
+	) -> DispatchResult {
+		match msg {
+			LiquidityPoolsMessage::AddCurrency { .. }
+			| LiquidityPoolsMessage::UpdateMember { .. }
+			| LiquidityPoolsMessage::AddPool { .. }
+			| LiquidityPoolsMessage::AddTranche { .. }
+			| LiquidityPoolsMessage::UpdateTrancheTokenPrice { .. } => {
+				<LiquidityPoolsGateway as OutboundQueue>::submit(sender, destination, msg)
+			}
+			_ => Err(DispatchError::Other("unsupported outbound message")),
+		}
+	}
+}
+
+parameter_types! {
+	pub const MaxIncomingMessageSize: u32 = 1024;
+}
+
+impl pallet_liquidity_pools_gateway::Config for Runtime {
+	type AdminOrigin = EnsureRootOr<TwoThirdOfCouncil>;
+	type InboundQueue = DummyInboundQueue;
+	type LocalEVMOrigin = pallet_liquidity_pools_gateway::EnsureLocal;
+	type MaxIncomingMessageSize = MaxIncomingMessageSize;
+	type Message = LiquidityPoolsMessage;
+	type Router = liquidity_pools_gateway_routers::DomainRouter<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeOrigin = RuntimeOrigin;
+	type WeightInfo = ();
+}
+
+/// DummyInboundQueue will be used in the first phase of testing in order to
+/// ensure that no incoming messages will be processed.
+pub struct DummyInboundQueue;
+
+impl InboundQueue for DummyInboundQueue {
+	type Message = pallet_liquidity_pools::Message<Domain, PoolId, TrancheId, Balance, Rate>;
+	type Sender = DomainAddress;
+
+	fn submit(_: Self::Sender, _: Self::Message) -> DispatchResult {
```

Comment:
Given that this dummy queue is an intermediary solution defined here, does it make sense to also publish events here? (Not sure if that's even possible tbh)

Reply:
If we throw independent of the sender or message, I don't see a reason to dispatch an event. I think it would be helpful to make the error message about more explicit, e.g. `InboundQueue not supported yet`.

Reply:
Not sure what we could add without bloating the message tbh.

Reply:
Can we just keep it like this since we will remove this eventually?